### ISSUE TITLE
Split within current vsplit

### ISSUE THREAD

**Issue by elzibubble** [chrisbra/NrrwRgn#64]
This could be an awful black morass but... I usually edit with 2-3 vsplits open:

	|a|b|c

The narrowed region opens above all these:

    narrow
	|a|b|c

which isn't very 'narrow'! I'd prefer:

	|a|  b   |c
	| |narrow|c

IE for my current vsplit to be hsplit. Hope that's clear!

I tried :belowright and :botright but neither seemed to do anything.

**Comment by chrisbra** [comment]
Yeah, custom commands did not respect the command modifiers like `:belowright` until recently. I'll see if I can implement this easily (at least for newer vims).
However in the meantime, have a look here:

https://github.com/chrisbra/NrrwRgn/blob/9a87f0c567c8dbc569154ed3c80e3b0a8972ecbd/doc/NarrowRegion.txt#L337-L345
